mac/Keyman4MacIM/Keyman4MacIM/InfoPlist.strings

```diff
@@ -15,6 +15,7 @@ CFBundleDisplayName = "Keyman";
  */
 
 //NSHumanReadableCopyright = "Copyright © 2018 SIL International. All rights reserved.";
-//CFBundleShortVersionString = "Keyman 10.0 for macOS";
+//CFBundleShortVersionString = "10.0";
```

Comment:
This information needs to come from the `/resources/VERSION.md` file.

Reply:
These are just comments. They *used* to be stuck in here, causing yet another step in every release to change them manually.
I figured by commenting them out, future maintainers wouldn't be tempted to put them back in. I could either take them out entirely or add a comment saying why they are NOT here and where they are: in the User-Defined section of Build Settings.  (and ONLY there, I hope)
There's also a Run Script that checks that the version matches resources/VERSION.md, failing the build if it doesn't match.
I am not confident about hacking a script that would write that value into Apple's xcodeproject file format externally. I figure a script to automatically remind you once per version change was a better compromise.

Reply:
Okay, perhaps changing these to reflect the place where they need to be updated is better.

I am okay with a test that fails if the version number mismatches, for now.

Reply:
Note added to this file. Pushed.